lib/hello_web/views/hello_view.ex

```diff
@@ -0,0 +1,3 @@
+defmodule HelloWeb.HelloView do
```

Comment:
@JuanMaGiraldo what a `View`does?

Reply:
It defines the view layer of the application. View has two functions:
- Render templates.
- View logic: functions that can transform and return data like decorators in RoR.